draft-ietf-quic-recovery.md

```diff
@@ -1153,16 +1142,13 @@ detected. Starts a new recovery period and reduces the congestion window.
 
 ### Process ECN Information
 
-Invoked when an ACK_ECN frame is received from the peer.
+Invoked when a packet is newly acknowledged.  This triggers a new congestion
+event if the peer indicates that the packet was ECN-CE marked.
 
 ~~~
-   ProcessECN(ack):
-     // If the ECN-CE counter reported by the peer has increased,
-     // this could be a new congestion event.
-     if (ack.ce_counter > ecn_ce_counter):
-       ecn_ce_counter = ack.ce_counter
-       // Start a new congestion event if the last acknowledged
-       // packet is past the end of the previous recovery epoch.
+   ProcessECN(packet):
```

Comment:
This is now simple enough I believe it'd be easier to read inlined and call CongestionEvent() in OnPacketAcked.

Reply:
Also, please update the Recovery Period section from "Recovery is a period of time beginning with detection of a lost packet or an increase in the ECN-CE counter." to "Recovery is a period of time beginning with detection of a lost  packet or receipt of an ECN-CE marked packet."  Or something similar

Reply:
And at the end of that paragraph "new losses or increases in the ECN-CE counter." -> " new losses or ECN-CE marks."